src/debian/control

```diff
@@ -30,8 +32,18 @@ Depends: ${python3:Depends},
          ${misc:Depends},
          python3-serial,
          python3-pil
+Suggests: python-fingerprint-doc
 Architecture: all
 Description: Python 3 written library for using ZhianTec fingerprint sensors
  Library for using ZhianTec ZFM-20, ZFM-60, ZFM-70 and ZFM-100 fingerprint
  sensors (known as "Arduino fingerprint sensor") in Python 3. Some other models
  like R302, R303, R305, R306, R307, R551 and FPM10A also work.
+
+Package: python-fingerprint-doc
+Section: doc
+Architecture: all
+Depends: ${sphinxdoc:Depends}, ${misc:Depends}
+Recommends: python-fingerprint | python3-fingerprint
+Description: API Documentation for PyFingerprint
```

Comment:
I think "API documentation" (lowercase letters) is more correct.

Reply:
Sure, I will do this